### PR TITLE
BCN - Mark and Jordi

### PR DESCRIPTION
**Pull request by markvertra**
@jviles

### CODE REVIEW

File: starter-code/views/index.ejs [hunk 1]
```diff
@@ -1,10 +1,13 @@
+<<<<<<< HEAD
```
Comment (tawebbcn) [inline starter-code/views/index.ejs:1]: If you have a conflict, you need to solve it. Now you have some weird HEAD snake attacking Messi.

File: starter-code/public/stylesheets/style.css [hunk 2]
```diff
@@ -0,0 +1,121 @@
+* {
+    box-sizing: border-box;
+    color: #1B5E20;
+}
+
+<<<<<<< HEAD
```
Comment (tawebbcn) [inline starter-code/public/stylesheets/style.css:6]: Merge conflict pending to be solved.

File: starter-code/views/random.ejs [hunk 3]
```diff
@@ -0,0 +1 @@
+<p><%= value %><p>
```
Comment (tawebbcn) [inline starter-code/views/random.ejs:1]: This code es exactly the same that the one in joke-by-category.ejs. Remember DRY. Use only one file (e.g. show-joke.ejs) or do different things.

**Comment by tawebbcn** [comment]
Be careful with the styles. The way you organized your code, the css is not available for all the routes. Check out what happens when you enter a joke category. You can learn more here: http://www.fullstacktraining.com/articles/how-to-serve-static-files-with-express

That said, great job! :1st_place_medal:

**Comment by markvertra** [comment]
Thanks!!!! This is going to be so useful :).

**Pull request closed without merging**